PACKAGES.md

```diff
@@ -29,11 +49,11 @@ They must be published and bundled in a traditional (transpiled) way.
 The bundle of these packages must work on both CommonJS and ECMAScript Module (ESM) environments.
 
 
-## Utility packages
+## Utilities packages
 
--   `@plone/drivers`
--   `@plone/helpers`
 -   `@plone/providers`
+-   `@plone/helpers`
+-   `@plone/drivers`
```

Comment:
Can you leave these sorted alphabetically?

Reply:
I've ordered by relevance to me, due to the current state of things (drivers and rsc are not a thing yet). It just occurred to me that if they do not exist yet, let's remove them.